changelog.html

```diff
@@ -150,6 +150,7 @@ <h1>User Import/Export Plugin Changelog</h1>
 
 <p><b>2.7.1</b> -- (tbd)</p>
 <ul>
+    <li>[<a href='https://github.com/igniterealtime/openfire-userImportExport-plugin/pull/1'>#1</a>] - Consolidate Maven project definition.</li>
```

Comment:
Wrong link here?

Reply:
sigh. It was the top issue, so I assumed it was number 1...